<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kuldeep" name="com.appmodule.testscript.Admin_Login_Script" tests="15" failures="15" timestamp="3 Jun 2016 03:55:57 GMT" time="0.032" errors="0">
  <testcase name="Click_on_Announcement" time="0.016" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Announcement -->
  <testcase name="Click_on_RollNumberSetup" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_RollNumberSetup -->
  <testcase name="Click_on_Message" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Message -->
  <testcase name="Click_on_Evaluation" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Evaluation -->
  <testcase name="Click_on_Attendance" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Attendance -->
  <testcase name="Click_on_Student" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Student -->
  <testcase name="Click_on_Teacher" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Teacher -->
  <testcase name="Click_on_BulkSection_SetUP" time="0.016" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_BulkSection_SetUP -->
  <testcase name="Click_on_Grades" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Grades -->
  <testcase name="Click_on_Events" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Events -->
  <testcase name="Click_on_ManageUser" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_ManageUser -->
  <testcase name="Click_on_Competency" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Competency -->
  <testcase name="Click_on_Gallery" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Gallery -->
  <testcase name="Click_on_BatchReport" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_BatchReport -->
  <testcase name="Click_on_Wall" time="0.000" classname="com.appmodule.testscript.Admin_Login_Script">
    <failure type="java.io.FileNotFoundException" message="C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\ykuld_000\Desktop\smartclassonlinesel\FlipLearn\Results\snapshots\Admin_Execution_Results.xlsx\Teacher_Execution_Results.xlsx (The system cannot find the path specified)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.appmodule.Generic_method.Generic_class.fn_CreateAndFormatExcel(Generic_class.java:321)
	at com.appmodule.Generic_method.Generic_class.fn_WriteResultToExcel(Generic_class.java:281)
	at com.appmodule.Generic_method.Generic_class.verifyUrl(Generic_class.java:360)
	at com.appmodule.testscript.Admin_Login_Script.loginapp(Admin_Login_Script.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- Click_on_Wall -->
</testsuite> <!-- com.appmodule.testscript.Admin_Login_Script -->
